src/Layout.soy

```diff
@@ -0,0 +1,17 @@
+{namespace Templates.Layout}
```

Comment:
We don't use this `Templates` prefix anymore, so this should just be `{namespace Layout}` now, similar to how we only do `{namespace Modal}` in [metal-modal](https://github.com/metal/metal-modal/blob/master/src/Modal.soy#L1).